<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainMenu</class>
 <widget class="QWidget" name="mainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="gameNameLabel">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>150</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ÎÒ</family>
     <pointsize>50</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>五子棋</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PVPButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>双人对战</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replayButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>复盘</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PVEButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>人机对战</string>
   </property>
  </widget>
  <widget class="QPushButton" name="guideButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>490</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>游戏说明</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>630</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeAccountButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>560</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>切换账号</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
